### ISSUE TITLE
Time to deprecate?

### ISSUE THREAD

**Issue by tobz** [tokio-rs/tokio-service#31]
Given that Tower duplicates/replicates a lot of the functionality expressed here, should we simply deprecate this crate?  Leave it up, but add the word `deprecated` somewhere in the repo description and/or README.

Nobody seems to ask about it in Gitter, and code-wise it's clearly stagnated.

/cc @carllerche

**Comment by tobz** [comment]
Actually, it looks like maybe just archiving the repo would be the right move.

**Comment by carllerche** [comment]
Yes, before we archive, could you submit a PR to this repo and tokio-proto w/ a big deprecated warning? Similar to this: https://github.com/carllerche/eventual/blob/master/README.md#this-crate-is-deprecated

tokio-service can point to tower. tokio-proto probably should point to the "what to do w/ tokio-proto" issue.
